src/openrct2-ui/input/Shortcuts.cpp

```diff
@@ -515,6 +517,31 @@ static void TileInspectorMouseDown(WidgetIndex widgetIndex)
     }
 }
 
+static void ShortcutToggleWallSlope()
+{
+    WindowBase* window = WindowFindByClass(WindowClass::TileInspector);
+    if (window == nullptr)
+    {
+        return;
+    }
+
+    const TileElement* tileElement = OpenRCT2::TileInspector::GetSelectedElement();
+
+    // Ensure an element is selected and it's a wall
+    if (tileElement == nullptr || tileElement->GetType() != TileElementType::Wall)
+    {
+        return;
+    }
+
+    int32_t currSlopeValue = tileElement->AsWall()->GetSlope();
+    int32_t newSlopeValue = (currSlopeValue + 1) % 3;
```

Comment:
Do we have some function somewhere that does this, to avoid the hardcoding?

Reply:
> Do we have some function somewhere that does this, to avoid the hardcoding?

The original functionality for changing the wall slope is based on the index of a dropdown menu. As a result, there doesn't appear to be any predefined variables or constants representing the different slope options or the total number of slope options. Should I add a local variable like:
`const int32_t totalSlopeOptions = 3;`
To make it clearer instead of using a magic number (3)?